android/src/main/java/com/google/android/cameraview/Camera1.java

```diff
@@ -34,12 +34,13 @@
 import java.io.IOException;
 import java.util.ArrayList;
 import java.util.List;
-import java.util.Objects;
 import java.util.Properties;
 import java.util.Set;
 import java.util.SortedSet;
 import java.util.concurrent.atomic.AtomicBoolean;
 
+import org.reactnative.camera.utils.ObjectUtils;
```

Comment:
you could also use this https://developer.android.com/reference/android/support/v4/util/ObjectsCompat